src/routes/auth/auth.service.ts

```diff
@@ -0,0 +1,156 @@
+import { Inject, Injectable, UnauthorizedException } from '@nestjs/common';
+import { VerifyAuthMessageDto } from '@/routes/auth/entities/schemas/verify-auth-message.dto';
+import { IAuthRepository } from '@/domain/auth/auth.repository.interface';
+import { IJwtService } from '@/datasources/jwt/jwt.service.interface';
+import { Request } from 'express';
+import {
+  CacheService,
+  ICacheService,
+} from '@/datasources/cache/cache.service.interface';
+import { CacheRouter } from '@/datasources/cache/cache.router';
+import { IConfigurationService } from '@/config/configuration.service.interface';
+import { SiweMessage } from '@/domain/auth/entities/siwe-message.entity';
+
+@Injectable()
+export class AuthService {
+  static readonly AUTH_TOKEN_TOKEN_TYPE = 'Bearer';
```

Comment:
We can increase encapsulation here and leave this field `private`.

Reply:
This value [is needed](https://github.com/safe-global/safe-client-gateway/blob/df2bc24a2eae01e4ac7007af40d2e1ac9d81c027/src/routes/auth/guards/auth.guard.ts#L37) by the `AuthGuard` in the follow up (#1369). Would you prefer I set it as `private` here then back?